config/deploy.rb

```diff
@@ -21,14 +21,16 @@
 set :db_remote_clean, true
 set :disallow_pushing, true
 
-set :sentry_api_token, ENV["SENTRY_AUTH_TOKEN"]
-set :sentry_organization, "resolve-to-save-lives"
-set :sentry_repo, "simpledotorg/simple-server"
-
 set :migration_command, "db:migrate:with_data"
 
-before "deploy:starting", "sentry:validate_config"
-after "deploy:published", "sentry:notice_deployment"
+if ENV["SENTRY_AUTH_TOKEN"]
```

Comment:
I think this is ok. Gives us the option of getting code to prod quickly in case of emergency, without having to deal with secondary issues like Sentry.

If we wanna restrict direct prod deploys from local environments, we can build specific logic for that, instead of piggybacking off of Sentry.

What do you think?

Reply:
There isn't a neat way to determine the deploy target's env. It's hard to say "mandatorily log for india, bd and sri lanka prod". Merging.